tools/shared.py

```diff
@@ -717,6 +717,7 @@ def exe_suffix(cmd):
 LLVM_NM=os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-nm')))
 LLVM_INTERPRETER=os.path.expanduser(build_llvm_tool_path(exe_suffix('lli')))
 LLVM_COMPILER=os.path.expanduser(build_llvm_tool_path(exe_suffix('llc')))
+LLD=os.path.expanduser(build_llvm_tool_path(exe_suffix('ld.lld')))
```

Comment:
On Gentoo Linux `llvm-*` is at `/usr/lib/llvm/7/bin` and `lld` is at `/usr/bin` btw.